homeassistant/components/trafikverket_train/config_flow.py

```diff
@@ -124,7 +124,7 @@ async def validate_input(
 class TVTrainConfigFlow(ConfigFlow, domain=DOMAIN):
     """Handle a config flow for Trafikverket Train integration."""
 
-    VERSION = 1
+    VERSION = 2
```

Comment:
Is it necessary to make this a major version change?

Reply:
Shouldn't be, it is backward compatible. So, this should indeed be a minor version bump, not a major one.